near/kycdao-ntnft/src/lib.rs

```diff
@@ -206,8 +334,13 @@ impl KycdaoNTNFT {
     }
 
     /// Check if the token is revoked
-    pub fn token_is_revoked(&self, token_id: TokenId) -> bool {
+    /*pub fn token_is_revoked(&self, token_id: TokenId) -> bool {
         self.token_statuses.get(&token_id).unwrap_or_default().is_revoked
+    }*/
+
+    /// Get the verification tier of a specific token
+    pub fn token_tier(&self, token_id: TokenId) -> String {
+        self.token_tiers.get(&token_id).unwrap_or("".to_string())
```

Comment:
Does it make sense to return the default if the token doesn't exist? There's no token doesn't exist error?

Reply:
Yeah, well we don't have an explicit token tier migration, so we are relying on this default instead. But I can add an explicit check for the token_id to exist at least, so at least it won't return anything for an invalid token ID. Good catch though.